fails to build, involves Bool.v and Nativevalues

Any ideas?

/Users/dan/src/coq-builder/encap-2/coq84hoq-8.4hoq/bin/coqtop -indices-matter -boot -nois -coqlib ./coq -R ./coq/theories Coq -compile coq/theories/Bool/Bool
File "/Users/dan/src/UnivalentFoundations/github/DanGrayson/HoTT/coq/theories/Bool/NCoq_Bool_Bool.ml", line 1, characters 0-17:
Error: Unbound module Nativevalues
Warning:
Removed file /Users/dan/src/UnivalentFoundations/github/DanGrayson/HoTT/coq/theories/Bool/Bool.vo
Anomaly: Library compilation failure. Please report.
Makefile:681: recipe for target 'coq/theories/Bool/Bool.vo' failed

--- a/.gitignore
+++ b/.gitignore
@@ -26,6 +26,8 @@ Makefile
 Makefile_ssrplugin
 # Ignore links into the original Coq library
 coq/dev
+coq/kernel
+coq/library
 coq/plugins
 # Ignore generated shell scripts and Makefile
 hoqtop

--- a/configure.ac
+++ b/configure.ac
@@ -176,8 +176,8 @@ AC_MSG_NOTICE([Creating symbolic links into Coq standard library])
 # We must have these links in the source directory, not in the build
 # directory, because the replacement standard library lives in the source
 # directory, and these links are required to make Coq accept it.
-rm -f $srcdir/coq/plugins $srcdir/coq/dev
-ln -s $COQLIB/plugins $COQLIB/dev $srcdir/coq
+rm -f "$srcdir/coq/dev" "$srcdir/coq/kernel" "$srcdir/coq/library" "$srcdir/coq/plugins"
+ln -s "$COQLIB/dev/" "$COQLIB/kernel/" "$COQLIB/library/" "$COQLIB/plugins/" "$srcdir/coq/"
 # TODO(jgross): Should we use AC_CONFIG_LINKS?  But $COQLIB/dev
 # doesn't exist in my installation, and that would make configure
 # error...

